api/batch/resource.go

```diff
@@ -57,7 +58,8 @@ const (
 
 var (
 	// Hardcoded based on python/batch-predictor/docker/base.Dockerfile
-	mainApplicationPath = "local:///home/spark/merlin-spark-app/main.py"
+	homeDir             = os.Getenv("IMG_BUILDER_PRED_HOME_DIR")
```

Comment:
It has to be part of the base image configuration for the prediction job since the `homeDir` will be specific to a docker image. i.e. previous release docker image expect `/spark/` but now it's `/home/spark` thus it won't work. It will be simpler if we keep it hardcoded to `/spark/